.github/workflows/centos8-ci.yml

```diff
@@ -0,0 +1,118 @@
+name: CI Centos8
```

Comment:
Can we share the code like ` .github/workflows/build.yml` instead of repeating centos7-ci?

Reply:
@spacewander Nice catch and that would be much better for multiple operating systems and architectures. I would make a change. Thanks.